### PR TITLE
feat: docs with welcome external links and faq

### PR DESCRIPTION
**Pull request by frederik-encord**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 <h1 align="center">
   <p align="center">Encord Active</p>
-  <a href="https://encord.com"><img src="src/encord_active/app/assets/encord_2_02.png" width="150" alt="Encord logo"/></a>
+  <a href="https://encord.com"><img src="docs/static/img/icons/encord_logo.png" width="150" alt="Encord logo"/></a>
```
Comment (Encord-davids) [inline README.md:11]: Nice one 💪

File: docs/docs/faq.mdx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+---
+
+## Initializing Encord Active is taking a long time, what should I do?
+
+For larger projects, initialization can take a while.
+While we're working on improving the efficiency, there are a couple of tricks that you can do.
+
+1. As soon as the metric metric computations have started (indicated by Encord Active printing a line containing `Running metric`) you can open a new terminal and run `encord-active visualize`.
+   This will allow you to continuously see what have been computed so far.
+   Do a browser refresh once in a while when new metrics are done computing in your first terminal.
```
Comment (Encord-davids) [inline docs/docs/faq.mdx:73]:
```suggestion
   Refresh the browser once in a while when new metrics are done computing in your first terminal.
```

File: docs/docs/faq.mdx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+[Here][finding-label-errors] is a blog post on how to find and fix label errors.
+
+---
+
+## How do I use Encord Active for active learning?
+
+Encord Active supports the active learning process by allowing you to
+
+1. Explore your data to select what to label next
+2. Finding label errors that potentially harm your model performance
```
Comment (Encord-davids) [inline docs/docs/faq.mdx:177]:
```suggestion
2. Find label errors that potentially harm your model performance
```

File: docs/docs/faq.mdx [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+3. Automatically decompose your model performance to help you determine where to put your focus for the next model iteration
+4. Tag subsets of data to set aside test sets for specific edge cases for which you want to maintain your model performance between each production model
+
+---
+
+## How do I upload my data and labels to Encord Annotate?
+
+Uploading your data to Encord Annotate is as simple as clicking the _Clone_ button on the [Filter and Export][filter-and-export] page.
```
Comment (Encord-davids) [inline docs/docs/faq.mdx:185]:
```suggestion
Uploading your data to Encord Annotate is as simple as clicking the _Export to Encord_ button on the [Filter and Export][filter-and-export] page.
```

**Review by Encord-davids — APPROVED** [review]

**Review by Encord-davids — APPROVED** [review]
